manifest.json

```diff
@@ -138,21 +138,21 @@
               "dockerfile": "1.1/jessie/sdk",
               "os": "linux",
               "tags": {
-                "1.1.5-jessie": {}
+                "1.1.5-1.1.6-jessie": {}
```

Comment:
The shared 1.1.5 tag should be updated to 1.1.5-1.1.6.

Reply:
Done. :)